e2e/node_test.go

```diff
@@ -162,6 +162,11 @@ func testNode() {
 				if dc == topolvm.DefaultDeviceClassAnnotationName {
 					continue
 				}
+				// The value will not match the overprovisioned capacity
+				// for thin pools.
+				if dc == "thin" {
```

Comment:
I understand that the device class of thinp can be skipped here, but I think that we need some equivalent tests for thinp.
I mean, in case of thinp, `capacity.topolvm.cybozu.com/<device-class>` is calculated as `(thin-pool-size * overprovisionedRatio)-(sum-of-all-thin-lv-virtual-sizes-in-thin-pool)`, and this calculation should be tested as well.

cf. https://github.com/topolvm/topolvm/blob/main/docs/proposals/thin-volumes.md#capacity-awareness

Reply:
There is currently no metric which provides this value. I am planning to add one and will include the test in that PR.

Reply:
Sounds great. Thank you.

Reply:
https://github.com/topolvm/topolvm/pull/537 is a draft PR for the new metric. Once this PR is merged I will update the draft to include this test.